core/src/main/java/org/infinispan/commands/GlobalRpcCommand.java

```diff
@@ -5,8 +5,10 @@
 import org.infinispan.factories.GlobalComponentRegistry;
 
 /**
- * Commands correspond to specific areas of functionality in the cluster, and can be replicated using the
- * {@link org.infinispan.remoting.inboundhandler.GlobalInboundInvocationHandler}.
+ * Commands correspond to specific areas of functionality in the cluster, and can be replicated using the {@link
+ * org.infinispan.remoting.inboundhandler.GlobalInboundInvocationHandler}. Implementations of this interface should not
```

Comment:
"should" is not strong enough IMO, I would say 

```
{@code @Inject} annotations on implementations will be ignored, components must be accessed via the
{@link GlobalComponentRegistry} parameter of {@link #invokeAsync(GlobalComponentRegistry)}.
```

Also please start on a new line ;)